hud_teaminfo.c

```diff
@@ -629,10 +666,26 @@ void Parse_TeamInfo(char *s)
 	ti_clients[client].org[2] = atoi(Cmd_Argv(4));
 	ti_clients[client].health = atoi(Cmd_Argv(5));
 	ti_clients[client].armor = atoi(Cmd_Argv(6));
-	ti_clients[client].items = atoi(Cmd_Argv(7));
+	ti_clients[client].items = Filter_FlagsAndRunes(client, atoi(Cmd_Argv(7)));
 	strlcpy(ti_clients[client].nick, Cmd_Argv(8), TEAMINFO_NICKLEN); // nick is optional
 }
 
+void Update_FlagStatus(int player_num, char *team, qbool got_flag)
+{
+	int flag = IT_KEY1 | IT_KEY2;
+	if (!strcmp(team, "blue")) {
```

Comment:
Need to revisit this tonight, might have mixed up the teams vs it_key's... in practice it doesn't matter as it's reversed in both mapping and UI condition as the correct flag char/icon is shown, but would be nice if the code makes complete sense. :)